spec/policies/user_policy_spec.rb

```diff
@@ -5,7 +5,7 @@
 describe UserPolicy do
   subject { described_class }
 
-  let(:lead_school) { create(:school, :lead) }
+  let(:school) { create(:school, :lead) }
```

Comment:
I think we might need some thought around the concept of still having lead schools like this. Not for this PR though.